site/docs/v1/tech/apis/messengers/_twilio-response-body.adoc

```diff
@@ -0,0 +1,37 @@
+==== Response Body
+
+[.api]
+[field]#messenger.accountSID# [type]#[String]#::
+The Twilio Account ID to use when connecting to the Twilio API. This can be found in your Twilio dashboard.
```

Comment:
@mooreds as mentioned in another comment, I copied most of the Twilio descriptions from the existing integrations docs.  I agree that I don't think the "This can be found in your Twilio dashboard" can be dropped unless there are any reservations given it existed before.

Reply:
Let's drop it from the 'response' then. Still probably useful on the 'request'